sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -633,6 +633,23 @@ case class SetNamespaceProperties(
     copy(namespace = newChild)
 }
 
+/**
+ * The logical plan of the ALTER NAMESPACE UNSET PROPERTIES command.
+ *
+ * The syntax of this command is:
+ * {{{
+ *     ALTER (DATABASE|SCHEMA|NAMESPACE) ... UNSET (DBPROPERTIES|PROPERTIES) [IF EXISTS] ...;
+ * }}}
+ */
+case class UnsetNamespaceProperties(
+    namespace: LogicalPlan,
+    propertyKeys: Seq[String],
+    ifExists: Boolean) extends UnaryCommand {
```

Comment:
We can probably use `RunnableCommand` to simplify the v2 command implementation. We can add a `UnaryRunnableCommand`, and then do
```
case class UnsetNamespacePropertiesCommand(...) extends UnaryRunnableCommand {
  ...
  def run(...) {
    val ResolvedIdentifier(catalog, ident) = child
    ...
  }
}
```

Reply:
Okay, let me give it a try, thanks.

Reply:
@cloud-fan I have improved it in the way you suggested.